File: ActividadesT2/ejercicios2.1/BUCLES 2.2/ejercicio24.py
def primo(numero):
    for i in range (2, numero):
        if (numero%i) == 0:
            return False
        else:
            return True

numero = 1
numeros = []
cant_primos = 0

while numero != 0:
    numero = int(input("Introduce un número mayor que 1 (0 para finalizar): "))
    
    if primo(numero) == True:
        numeros.append(numero)

cant_primos = len(numeros)

print(f"La cantidad de números primos introducidos son: {cant_primos}")

File: Kotlin/PRIMERA RELACIÓN DE EJERCICIOS AFRICA/BUCLES/src/Ejercicio24.kt
/* Escribir un programa que solicite el ingreso de una cantidad indeterminada de números
mayores que 1, finalizando cuando se reciba un cero. Imprimir la cantidad de números
primos ingresados */

fun esPrimo(n: Int): Boolean {
    if (n <= 1) return false
    for (i in 2 until n) {
        if (n % i == 0) {
            return false
        }
    }
    return true
}

fun main(){
    var primos = 0
    do {
        print("Introduce un numero mayor que 1: ")
        var num = readLine()?.toIntOrNull() ?: 1
        if(esPrimo(num)){
            primos++
        }
    } while(num != 1)
    println("Has introducido ${primos} números primos")
}